Civi/Core/SettingsMetadata.php

```diff
@@ -69,9 +69,7 @@ public static function getMetadata($filters = [], $domainID = NULL, $loadOptions
     }
 
     self::_filterSettingsSpecification($filters, $settingsMetadata);
-    if ($loadOptions) {
```

Comment:
@colemanw so this seems to result in us iterating through every setting and doing noting if loadOptions is false - as opposed to just skipping. Why the change?

Reply:
Also - I note loadOptions is not cached so that doesn't mitigate the empty iteration

Reply:
You're right. I originally had this doing more but had to change it back due to a failing test.

Reply:
@colemanw is it going to do more again soon? If so I guess it's Ok to merge as is

Reply:
Probably not. I've replaced the conditional as before.